governance.md

```diff
@@ -18,88 +36,180 @@ The committee MUST:
 * Provide designated alternates in cases where quorum is required but not attainable with the current set of members
 * Communicate with the Continuous Delivery Foundation on a regular cadence
 
-## Committee Deliverables
+#### Former members ❤️
```

Comment:
Do we need to state some connection between Emeritus and Former members or is that self explanatory for non native English speakers?

Reply:
If it was not clear I will add a comment about this, sorry about the confusion

Reply:
Added a comment at the end of "processes.md"

Reply:
Aha, this was about the former Governing board members.